class.rb

```diff
@@ -0,0 +1,23 @@
+require_relative './enumerable'
+
+class MyList
+  include MyEnumerable
+
+  def initialize(*list)
+    @list = list
+  end
+
+  def each
+    (0...@list.length).each do |i|
+      yield @list[i]
+    end
+  end
```

Comment:
- [ ] Try to avoid using the existing #each method in the each method you are creating, you can try to use a while loop for example.